### ISSUE TITLE
Bug: `sort-imports` doesn't pick up `internalPattern`

### ISSUE THREAD

**Issue by selfagency** [azat-io/eslint-plugin-perfectionist#260]
### Describe the bug

I'm using Sveltekit, which provides `$lib` as an alias for easy access to the library folder. I'm using it all throughout my project. I specified in my rules:

```js
      'perfectionist/sort-imports': [
        'error',
        {
          internalPattern: ['$lib/**'],
          type: 'natural',
        },
      ],
```

However, it continues treating `$lib` as an external module.

### Code example

```
import { cart } from '$lib/services/stores';
import { assign } from 'radashi';
```

### ESLint version

v9.9.1

### ESLint Plugin Perfectionist version

v3.4.0

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Hi!

I have not been able to reproduce the problems. Tried to do it locally, everything works correctly for me.

Can you create a repository to reproduce the problem?

**Comment by selfagency** [comment]
I had it misconfigured. False alarm.

**Comment by joakim** [comment]
For the record, this is the correct regex pattern to match `$lib/**`:

```js
'perfectionist/sort-imports': [
  'error',
  {
    internalPattern: ['^\\$lib/.+']
  },
],w
```
